tests/trace/example.py

```
@@ -444,6 +445,27 @@
+    )
+    weight_prepared, scale_prepared = flashinfer.prepare_mxfp8_trtllm_weights(
+        weight_mxfp8,
+        weight_scale,
+    )
+    flashinfer.mm_mxfp8_dynamic_quant(
+        a_bf16,
+        weight_prepared,
+        scale_prepared,
+    )
+except Exception:
+    pass  # Requires Blackwell (SM100/103/107)
```

Ruff (0.16.2)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)